integrations/cloud-notifications/integrations/pagerduty.md

```diff
@@ -32,7 +32,7 @@ Steps to configure your PagerDuty to receive notifications from Netdata:
 
 1. Create a service to receive events from your services directory page on PagerDuty
 2. At step 3, select `Events API V2` Integration or **View Webhooks** if you already have some defined
-3. Once the service is created, you will be redirected to its configuration page, where you can copy the **Integration Key** and **Integration URL (Alert Events)** fields to add them to your notification configuration in the Netdata UI.
+3. Once the service is created you will be redirected to its configuration page, where you can copy the **integration key**, that you will need need to add to your notification configuration on Netdata UI.
```

Comment:
@Ancairon, hey. Do you have any idea why this changed? It is [different in the master](https://github.com/netdata/netdata/blob/2ceead0b1877a5f0cebe254bde6e16049d96b2ef/integrations/cloud-notifications/integrations/pagerduty.md?plain=1#L35).